src/components/safe-apps/AppCard.tsx

```diff
@@ -24,20 +25,22 @@ type AppCardContainerProps = {
 export const AppCardContainer = ({ url, children }: AppCardContainerProps): ReactElement => {
   if (url) {
     return (
-      <a href={url} target="_blank" rel="noreferrer">
-        <Card
-          sx={({ palette }) => ({
-            height: 190,
-            transition: 'background-color 0.3s ease-in-out, border 0.3s ease-in-out',
-            '&:hover': {
-              backgroundColor: palette.primary.background,
-              border: `2px solid ${palette.primary.light}`,
-            },
-          })}
-        >
-          {children}
-        </Card>
-      </a>
+      <Link href={url}>
+        <a rel="noreferrer">
```

Comment:
we don't need the `a` anymore because of `Link`, do we?

Reply:
next link use to have the `a`  inside => https://nextjs.org/docs/api-reference/next/link 

In this case is keeping the mouse pointer for example